src/devices/Ina226/README.md

```diff
@@ -0,0 +1,124 @@
+﻿# Ina226 - Bidirectional Current/Power Monitor w/ Alert
+
+## Summary
+
+The INA226 is a current shunt and power monitor with an I2C or SMBUS-compatible
+interface. The device monitors both a shunt voltage drop and bus supply voltage.
+Programmable calibration value, conversion times, and averaging, combined with
+an internal multiplier, enable direct readouts of current in amperes and power
+in watts.
+
+* Senses Bus Voltages from 0 to 36 V
+* High-Side or Low-Side Sensing
+* Reports Current, Voltage, and power
+* High Accuracy: 0.1% Gain Error (Max) 10uv Offset(Max)
+* Configurable Averaging Options
+* 16 Programmable Addresses
+
+## Documentation
+
+* [INA226 Datasheet](https://www.ti.com/lit/ds/symlink/ina226.pdf)
```

Comment:
Please remove the PDF from the PR, we can't include this for copyright reasons.

Reply:
Noted for future reference and removed.